iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/FileUploadTests.java

```diff
@@ -508,19 +409,15 @@ public void uploadToBlobAsyncMultipleFilesParallel() throws URISyntaxException,
         for (int i = 1; i < MAX_FILES_TO_UPLOAD; i++)
         {
             final int index = i;
-            executor.submit(new Runnable()
+            executor.submit(() ->
```

Comment:
`() ->` !! and I thought I have seen it all :D

Reply:
Pretty fancy stuff, yeah!